vault.py

```diff
@@ -74,22 +74,30 @@ def run(self, terms, inject=None, variables=None, **kwargs):
             raise AnsibleError('Vault authentication token missing. Specify with'
                                ' VAULT_TOKEN environment variable or $HOME/.vault-token')
 
+        cafile = os.getenv('VAULT_CACERT')
+        capath = os.getenv('VAULT_CAPATH')
         try:
-            context = ssl.create_default_context(cafile=os.getenv('VAULT_CACERT'),
-                                                 capath=os.getenv('VAULT_CAPATH'))
+            if cafile or capath:
+                context = ssl.create_default_context(cafile=cafile, capath=capath)
+            else:
+                context = None
             request_url = urljoin(url, "v1/%s" % (key))
             req = urllib2.Request(request_url, data)
             req.add_header('X-Vault-Token', token)
             req.add_header('Content-Type', 'application/json')
-            response = urllib2.urlopen(req, context=context)
+            if context:
+                response = urllib2.urlopen(req, context=context)
+            else:
+                response = urllib2.urlopen(req)
         except AttributeError as e:
             python_version_cur = ".".join([str(version_info.major),
                                            str(version_info.minor),
                                            str(version_info.micro)])
             python_version_min = "2.7.9"
             if StrictVersion(python_version_cur) < StrictVersion(python_version_min):
                 raise AnsibleError('Unable to read %s from vault:'
-                                   ' Using Python %s and vault lookup plugin requires at least %s'
+                                   ' Using Python %s, and vault lookup plugin requires at least %s'
+                                   ' to use an SSL context (VAULT_CACERT or VAULT_CAPATH)'
```

Comment:
If you don't want to accept the modified form from https://github.com/jhaals/ansible-vault/pull/26/commits/b73fb191846ad32d7612936b6c7f822d6131349f, and revert back to the original (see line 84 on the left here), then this change narrowing the error case will also need to be reverted.